### PR TITLE
build: unify ui and ui-server charts

### PR DESCRIPTION
**Pull request by ciyer**
Combine the charts into one chart for the UI.

/deploy renku-ui=master #persist

### CODE REVIEW

**Comment by rokroskar** [comment]
@lorenzo-cavazzi can we deploy this? I guess it needs a new values file...

**Comment by lorenzo-cavazzi** [comment]
Iirc the CI deployments already have the required values to deploy it.
I just triggered a deplyoment

**Comment by rokroskar** [comment]
? wouldn't that mean that other CI deployments fail since this isn't merged yet?

**Comment by lorenzo-cavazzi** [comment]
I suppose the CI values file currently has both the old values (`ui` and `uiserver`) and the new values (`ui.server` and `ui.client`)

**Comment by rokroskar** [comment]
ha, you're probably right... that possibility has never occurred to me 🙈

**Comment by lorenzo-cavazzi** [comment]
I guess the counts as a reminder to clean up the unnecessary values after merging the PR :grin:

File: helm-chart/values.yaml.changelog.md [hunk 1]
```diff
@@ -6,6 +6,23 @@ Please follow this convention when adding a new row
 * `<type: NEW|EDIT|DELETE> - *<resource name>*: <details>`
 
 ----
+## Upgrading to Renku 0.13.1
```
Comment (rokroskar) [inline helm-chart/values.yaml.changelog.md:9]:
```suggestion
## Upgrading to Renku 0.15.0
```

**Review by rokroskar — CHANGES_REQUESTED** [review]

**Comment by lorenzo-cavazzi** [comment]
Already merged through #2665

**Pull request closed without merging**